.github/workflows/eslint.yml

```
@@ -0,0 +1,33 @@
+name: Lint Frontend
+
+on:
+  pull_request:
+    branches:
+      - "main"
+      - "master"
+    types: [opened, synchronize, reopened]
+    paths:
+      - 'frontend/**'
+
+jobs:
+  lint:
+    name: Run ESLint for Frontend
+    runs-on: ubuntu-latest
+
+    defaults:
+      run:
+        working-directory: frontend
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 24
```

```suggestion
          node-version: 22
```
Is there any reason why we are not using the LTS version?

---

I used Node 24 since it's going to become the next LTS in October 2025. As you are suggesting, would it be better to use the current LTS (22) until October and update to 24  only once the switch happens in October?